src/storage-preview/azext_storage_preview/_transformers.py

```diff
@@ -122,5 +124,51 @@ def transform_storage_list_output(result):
     return list(result)
 
 
+# pylint: disable=unused-argument
 def transform_file_upload(result):
     return None
+
+
+def transform_queue_stats_output(result):
+    if isinstance(result, dict):
+        new_result = {}
+        from azure.cli.core.commands.arm import make_camel_case
+        for key in result.keys():
+            new_key = make_camel_case(key)
+            new_result[new_key] = transform_queue_stats_output(result[key])
+        return new_result
+    return result
+
+
+def transform_message_list_output(result):
+    for i, item in enumerate(result):
+        result[i] = transform_message_output(item)
+    return list(result)
+
+
+def transform_message_output(result):
+    result = dict(result)
+    from collections import OrderedDict
+    new_result = OrderedDict()
```

Comment:
May I know why you want to remember the order of dict items here?

Reply:
changed it to common dict